app/controllers/post.js

```diff
@@ -26,6 +26,15 @@ module.exports = {
       }
     })
   },
+  update: function (req, res, next) {
+    userModel.findByIdAndUpdate(req.params.id, { name: req.body.name, email: req.body.email, password: req.body.password }, function (err, result) {
```

Comment:
@tanujvyas10 instead of writing req.body all the time use destructuring 
like const  { email, name, password } = req.body;